linkml/generators/docgen.py

```diff
@@ -656,7 +656,7 @@ def all_class_objects(self) -> Iterator[ClassDefinition]:
         Ensures rank is non-null
         :return: iterator
         """
-        elts = self.schemaview.all_classes().values()
+        elts = self.schemaview.all_classes(imports=self.mergeimports).values()
```

Comment:
Is this necessary? Similar comment for the other elements, i.e., slots, enums, types, etc.

The reason I ask is because if you look at PR https://github.com/linkml/linkml-runtime/pull/233/files which added the ability to honor `--mergeimports`, it makes a call to `merge_imports()` which calls the closure that looks through all the imported schemas and fetches the classes, slots, enums, etc. appropriately and associates with the SchemaView object.

Reply:
So what I mean to say is that:

```python
from linkml_runtime.utils.schemaview import SchemaView

sv = SchemaView("~/path/to/linkml.yaml", merge_imports=True)

sv.all_classes()  # same result as below
sv.all_classess(imports=True)  # same result as above
```

Reply:
I will add new comment as I might have figured out the issue.